### PR TITLE
Mat 6584 creating email content

### PR DESCRIPTION
**Pull request by gregory-akins**
## PR Template

Jira Ticket: [MAT-6584](https://jira.cms.gov/browse/MAT-6584)
(Optional) Related Tickets:

### Summary
Formatted text email success & failures messages.  If the error message from parsing MatMeasure is a valid JSON, then I try to parse the JSON and look for the message; otherwise, just include whatever error text is available.

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included
* [X] No extemporaneous files are included (i.e. Complied files or testing results)
* [X] This PR is into the **correct branch**.
* [X] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/resultutils.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  status: string;
+  message: string | undefined;
+  validationErrors: any;
+}
+
+export const parseError = (message: string): string => {
+  try {
+    const err: ErrorMessage = JSON.parse(message);
+    if (err.message) {
+      console.log("########## undefined", err.message);
```
Comment (jkotanchik-SB) [inline lambda-function/src/utils/resultutils.ts]: What does 'undefined' mean here?

**Review by jkotanchik-SB — DISMISSED** [review]

File: lambda-function/src/utils/resultutils.ts (hunk 1 above)
Comment (gregory-akins) [reply lambda-function/src/utils/resultutils.ts]: Oops.. left some debuggign code.. also added a comment to make it more clear

**Review by jkotanchik-SB — DISMISSED** [review]

**Review by jkotanchik-SB — APPROVED** [review]
